config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "home#index"
```

Comment:
- [ ] We should have a users index route, not home index route. Kindly fix this. You should create the user index route and define it as the root path.

  ```
 root "users#index"
 ```